application/helpers/pager_helper.php

```diff
@@ -43,3 +43,37 @@ function pager($base_url, $model)
 
     return $pager;
 }
+
+/**
+ * Returns a printable pagination detailed
+ *
+ * @param $model
+ * @return string
+ */
+function pager_detailed($model)
+{
+    $CI = &get_instance();
+    
+    $current_item_last = $CI->$model->default_limit * $CI->$model->current_page;
+    $current_item_first = ($current_item_last - $CI->$model->default_limit) + 1;
+    $total_pages = ($CI->$model->total_pages > 0)?$CI->$model->total_pages:1;
+    $total_rows = $CI->$model->total_rows;
+    
+    if ($total_rows == 0) {
+        $current_item_last = 0;
+        $current_item_first = 0;
+    }
+    
+    $pager = '<div class="model-pager-detailed">';
+    $pager .= '<div>';
+    $pager .= trans('showing').' '.$current_item_first;
```

Comment:
You may add actual buttons for navigation here so the user does not have to scroll up to get to the next page

Reply:
It was not my initial idea. Only show information of the total of records and the existence of pages.